### DISCUSSION TITLE
基于开源技术栈的数据血缘、治理参考解决方案 - siwei.io

### DISCUSSION THREAD

**Discussion by giscus** [siwei-io/siwei-io.github.io#22] [Comments]
# 基于开源技术栈的数据血缘、治理参考解决方案 - siwei.io

也许我们没有必要从头在 NebulaGraph 上搭建自己的数据血缘项目，本文分享如何用开源、现代的 DataOps、ETL、Dashboard、元数据、数据血缘管理系统构建大数据治理基础设施

https://siwei.io/data-lineage-oss-ref-solution/

**Comment by zyw8136** [comment]
请问sample_postgres_loader_nebula.py这个python文件在哪里？

**Reply by wey-gu** [reply]
https://github.com/wey-gu/amundsen/tree/main/databuilder/example/scripts

**Reply by wey-gu** [reply]
我好像没把这个例子推到远端分支🤦

**Reply by zyw8136** [reply]
OK，能提交一下吗？

**Reply by zyw8136** [reply]
非常感谢！

**Reply by zyw8136** [reply]
我好像找了一个有这个文件的分支

**Reply by zyw8136** [reply]
https://github.com/wey-gu/amundsen/tree/amundsen_nebula_graph 应该是这个分支

**Reply by wey-gu** [reply]
@zyw8136

在这个分支哈，我找了一下，发现已经推上去了

https://github.com/wey-gu/amundsen/tree/amundsen_nebula_graph/databuilder/example/scripts

**Reply by wey-gu** [reply]
抱歉之前贴错了 branch

**Reply by wey-gu** [reply]
奥奥，对对，才看到你已经找到了。

**Reply by zyw8136** [reply]
没有关系，还是非常感谢您的分享。另外还一个地方：meltano invoke dbt docs to serve多了一个to；应该是meltano invoke dbt docs serve

**Reply by wey-gu** [reply]
感谢，回头我 fix 一下哈，另外，请注意我的 RFC 还没有被 amundsen 社区接受，代码还只是 PoC 阶段哈。

https://github.com/amundsen-io/rfcs/pull/48

**Reply by zyw8136** [reply]
OK，另外在superset的meltano.yml里面，少了对superset的下载设置，总是skip installing

**Reply by wey-gu** [reply]
感谢！
